theme/common.less

```
@@ -121,6 +121,43 @@ pre {
     }
 }
 
+.zoomSliderBar {
+    -webkit-appearance: none;  /* Override default CSS styles */
+    appearance: none;
+    width: 100%; /* Full-width */
+    height: .5rem;
+    background: black; /* Grey background */
```

use less variable so target themes can override

```suggestion
    background: @black; /* Grey background */
```